### PR TITLE
[CFM-1217] Adding the comments to "My content" view

### PR DESCRIPTION
**Pull request by ordavidil**
https://issuetracker.amplexor.com/jira/browse/CFM-1217

### CODE REVIEW

**Comment by ordavidil** [comment]
@soniCaH 
I'm having some difficulties to add the comments as contextual filter to the view itself, I could use your help in here.
Thanks.

File: project/profiles/capacity4more/modules/c4m/search/c4m_search/c4m_search.features.inc [hunk 1]
```diff
@@ -380,6 +380,115 @@ function c4m_search_default_search_api_index() {
     "entity_type" : "search_api_index",
     "rdf_mapping" : []
   }');
+  $items['search_comments'] = entity_import('search_api_index', '{
```
Comment (soniCaH) [inline project/profiles/capacity4more/modules/c4m/search/c4m_search/c4m_search.features.inc]: Could you please rename it to c4m_search_comments to be consistent with the other indexes?

**Comment by soniCaH** [comment]
Hey @ordavidil 

I think we only need to change the migrate script:

`migrate.sh`:
```
function pre_migrate {
  // ...
  drupal_drush --uri="$SITE_URL" search-api-disable -y c4m_search_nodes
  drupal_drush --uri="$SITE_URL" search-api-disable -y c4m_search_users
 // Disable comment index
}

function post_migrate {
  drupal_drush --uri="$SITE_URL" search-api-enable -y c4m_search_nodes
  drupal_drush --uri="$SITE_URL" search-api-enable -y c4m_search_users
  // Enable comment index
  
  // ...
  
  drupal_drush --uri="$SITE_URL" search-api-index c4m_search_nodes && drupal_drush --uri="$SITE_URL" search-api-index c4m_search_users
  // Add comment index

  // ...
}
```

I did not test it yet, but https://www.drupal.org/project/search_api_multi seems like a solution for the view with multiple indexes.

> When the module is enabled, you'll get a new Views base table where you can search all enabled Search API indexes at once.

If this does not work with the facets, we might have to create a new page `user/%/content` and `user/%/comment` for example and link between both.

**Comment by ordavidil** [comment]
@soniCaH 
please check this out

### My content view
selection_338

### My comments view

selection_339

**Pull request merged**